### DISCUSSION TITLE
Does huly support desktop application?

### DISCUSSION THREAD

**Discussion by perperj-super** [hcengineering/platform#5652] [Q&A] [answered]
Hello!

Just want to know if huly supports desktop application.

**Comment by aplatoff** [comment, marked as answer]
Actually we have. MacOS and Linux will be available on the week of Jun 17th 2024. Windows later, the only problem to solve there is to sign app with Microsoft.

**Reply by aquilaplus** [reply]
Can you share a direct link to the desktop app ? The app seems available only the US Mac Appstore but I can't find a direct link to download it.

Thanks !

**Comment by chirag-mtpl2023** [comment]
If you are going with a native tech-stack then you have to develop separate platforms like Mac, Windows, and Linux, But If we are going with a Hybrid solution then we can build a robust solution for all 3 platforms.

**Comment by nyxb** [comment]
its svelte https://tauri.app/ btw for all linux windows, mac and ios android
